code/modules/surgery/tools.dm

```diff
@@ -247,6 +247,8 @@
 	. = ..()
 	if(!proximity)
 		return
+	if(!isitem(I))		//Check if it is an actual item
+		return
```

Comment:
I don't think this check is needed. There's an item type check just a few lines below.

Reply:
If you mean this `if(!isorgan(I) && !isbodypart(I))` then yes, but if you click on anything other than an organ or a bodypart, like a machine or something then it will always show the message that the bag can only hold bodyparts.

Reply:
1. The code comment doesn't describe your intention.
2. I am not convinced of your intention.

Reply:
I made this check because currently if you click on literally anything like walls, tiles, machines etc. the other checks _will_ trigger because either the bag is full or the thing is not an organ or a bodypart. I added the check specifically to check if the thing is an item to reduce the cases where this will happen, that is also why this was above both of the checks and not only above the `if(!isorgan(I) && !isbodypart(I))` check.

But i removed it entirely since it is not _that_ needed.